Purchases/Logging/Strings/PurchaseStrings.swift

```diff
@@ -41,6 +41,14 @@ enum PurchaseStrings {
     case requested_products_not_found(request: SKRequest)
     case callback_not_found_for_request(request: SKRequest)
     case unable_to_get_intro_eligibility_for_user(error: Error)
+    case duplicate_refund_request(details: String)
```

Comment:
i was thinking these cases would be good to create suberrors for...anyone have thoughts?

Reply:
I think adding in suberrors makes a lot of sense if these errors can fit under an umbrella error that is surfaced to the user. That also means we wouldn't have to update our public Error API footprint.

Reply:
Well, all of these strings are currently being used as the messages for the umbrella BeginRefundRequestError. I was thinking they might be better as suberrors...

But I'm realizing we need to merge main into storekit_2 before that will be possible. I can start working on that now